testing/common_testing_util.py

```diff
@@ -22,18 +23,25 @@
 
 path_repo_root: Path = Path(__file__).parents[FOLDER_LEVELS_FROM_ROOT].absolute()
 
-path_temp: str = str(path_repo_root / "tmp")
+if not sys.platform.startswith('win'):
+    path_temp: Path = Path("/tmp")
+else:
+    path_temp: Path = Path(Path.home().parts[0], 'temp')
```

Comment:
What about using MKDTemp instead? https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp

Those will persist in respective OS temp and stick around until deleted by user specifically.

Reply:
Even better [`TemporaryDirectory`](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory):

```suggestion
with tempfile.TemporaryDirectory() as path_temp:
```

this form deletes itself when done.

Reply:
Yeah... one of the goal was to keep a limited number of these directories for comparisons and tracking unit tests outputs history.